GentApp/Views/Company/AddEventPage.xaml.cs

```diff
@@ -75,10 +63,20 @@ private void validateInput()
 				EndDateValidationErrorTextBlock.Text = "This field is required.";
 				isValid = false;
 			}
-			if (isValid == true)
+			if ( isValid )
 			{
-				Event newEvent = new Event() { Title = Title.Text, Description = Description.Text, StartDate = StartDatePicker.Date.Value.DateTime, EndDate = EndDatePicker.Date.Value.DateTime, BranchId = SimpleIoc.Default.GetInstance<CompanyViewModel>().SelectedBranch.Id};
-				SimpleIoc.Default.GetInstance<BranchViewModel>().AddEvent(newEvent);
+				Event newEvent = new Event() {
+					Title = Title.Text,
+					Description = Description.Text,
+					StartDate = StartDatePicker.Date.Value.DateTime,
+					EndDate = EndDatePicker.Date.Value.DateTime };
+				SimpleIoc.Default.GetInstance<CompanyViewModel>().SelectedBranch.Events.Add(newEvent);
+				var branch = SimpleIoc.Default.GetInstance<CompanyViewModel>().SelectedBranch;
+				var company = SimpleIoc.Default.GetInstance<CompanyViewModel>().MyCompany;
+
+				SimpleIoc.Default.GetInstance<CompanyViewModel>().MyCompany.Branches[company.Branches.FindIndex(i => i.Equals(branch))] = branch;
+
+				SimpleIoc.Default.GetInstance<CompanyViewModel>().SaveCompanyCommand.Execute(null);
```

Comment:
wot

Reply:
da snap ik ni, twerkt wel, leg mij keer uit haha

Reply:
ik set de branch daje wil bewerken gelijk aan de nieuwe branch met het nieuwe event in de company van de user
en dan save ik de company

Reply:
aha ok